draft-irtf-cfrg-vdaf.md

```diff
@@ -1365,7 +1365,7 @@ def ping_pong_transition(
             agg_param: Vdaf.AggParam,
             prep_shares: Vec[bytes],
             prep_state: Vdaf.PrepState,
-         ) -> (State, Optional[Message]):
```

Comment:
`ping_pong_continued`, below, returns `(State, Optional[Message])` and does `return Vdaf.ping_pong_transition(...)`. Do we have to do anything there to translate the `Message` value into something like `Some(message)`? Or is returning non-`None` good enough?